src/sentry/conf/server.py

```diff
@@ -145,9 +145,17 @@ def env(key, default="", type=None):
         "PORT": "",
         "AUTOCOMMIT": True,
         "ATOMIC_REQUESTS": False,
-    }
+    },
+    "metrics-spanner": {
+        "ENGINE": "django_spanner",
+        "PROJECT": "search-and-storage",
+        "INSTANCE": "markus-test-spanner",
+        "NAME": "markus-test-spanner-db",
+    },
```

Comment:
Do you expect to move this out of the self hosted settings ?

Reply:
I haven't thought about it yet. Perhaps we can move this database config into getsentry entirely again, where the db router also already exists.